trading_cost.py

```diff
@@ -0,0 +1,64 @@
+"""
+A file to build cost objects.
+"""
+
+class CurrencyCost:
+    """
+    Save information in a currency trading.
```

Comment:
This is description of this object. What you write now is not "what this object is, what it is for."
It should be something like, for example,
`Cost function of currency trading.`